frontend/lib/src/components/widgets/NumberInput/NumberInput.tsx

```diff
@@ -467,6 +467,8 @@ export const NumberInput: React.FC<Props> = ({
                 step: step,
                 min: min,
                 max: max,
+                type: "number",
```

Comment:
nit: since we don't have an e2e test for mobile devices to enforce this automatically, could you add a comment here explaining that this is added to show a numeric keyboard on mobile devices, please?

Reply:
Comment added!